ci-operator/step-registry/ipi/aws/pre/edge-zones/opt-in/ipi-aws-pre-edge-zones-opt-in-ref.yaml

```diff
@@ -11,12 +11,6 @@ ref:
       cpu: 10m
       memory: 100Mi
   env:
-  - name: EDGE_ZONE_TYPE
```

Comment:
Hello @mtulio, As discussed in https://github.com/openshift/release/pull/47045#discussion_r1446297816, this PR is going to remove `EDGE_ZONE_TYPE`,  the `e2e-aws-ovn-edge-zones` [test](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/48418/rehearse-48418-periodic-ci-openshift-release-master-nightly-4.15-e2e-aws-ovn-edge-zones/1754365108290588672) get passed, PTAL, thanks.